lib/components/app/app-menu.js

```diff
@@ -65,13 +73,16 @@ class AppMenu extends Component {
 // connect to the redux store
 
 const mapStateToProps = (state, ownProps) => {
+  const {language, modules} = state.otp.config
   return {
-    languageConfig: state.otp.config.language
+    languageConfig: language,
+    mailablesEnabled: Boolean(modules?.find(m => m.id === 'mailables'))
```

Comment:
Can you put a commented-out example of the mailables module config in example.config?